documentation/faq/60-env-vars.md

```diff
@@ -2,20 +2,20 @@
 question: How do I use environment variables?
 ---
 
-Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed. You would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env['YOUR_ENV_VAR']`. We hope to see all environment variables exposed on the server-side [in the future](https://github.com/vitejs/vite/issues/3176).
+Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. By default, Vite only exposes environment variables prefixed with `VITE_`. However, SvelteKit overrides Vite's default `envPrefix` setting to also allow `PUBLIC_` as a prefix. You would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env['YOUR_ENV_VAR']`. We hope to see all environment variables exposed on the server-side [in the future](https://github.com/vitejs/vite/issues/3176).
```

Comment:
@dummdidumm I don't think this is breaking, but maybe it can be phrased more friendlier here

```suggestion
Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. SvelteKit allows you to use any environment variables prefixed with either `VITE_` or `PUBLIC_`. You would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env['YOUR_ENV_VAR']`. We hope to see all environment variables exposed on the server-side [in the future](https://github.com/vitejs/vite/issues/3176).
```

Reply:
Ah I misread the code - the allowed prefixes are now both vite and client, right?

Reply:
Yeap

Reply:
Wait no, it's vite and public

Reply:
I updated the text similar to your suggestion, but also mentioning `envPrefix` still

Correct that this allows both to avoid being breaking